db/migrate/20200423070105_create_lots.rb

```diff
@@ -0,0 +1,16 @@
+class CreateLots < ActiveRecord::Migration[6.0]
+  def change
+    create_table :lots do |t|
+      t.string :title, null: false
+      t.string :description
+      t.string :image, null: false
+      t.string :status
```

Comment:
Status is a string. Which status  can lot have? I remember that suggested you use enum for status.  https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html
And also you can use enum for state too